src/config-hardware/webcams.md

```diff
@@ -0,0 +1,23 @@
+# Configuring Webcams
+
+Pop!\_OS will automatically detect most USB webcam models upon connection.
```

Comment:
Rather than just having this vague statement, I would suggest mentioning or briefly explaining the V4L2/UVC driver model. (I saw that there's a two-sentence section on UVC below; it might be worth folding that into this introduction.)

> Pop!_OS will automatically detect any webcams or capture cards with drivers that implement the Video4Linux2 (V4L2) API. Most USB webcams are supported via the Universal Video Class (UVC) driver built into the Linux kernel. A non-exhaustive list of compatible webcams can be found on the [UVC driver's website](https://www.ideasonboard.org/uvc/).

I changed the link from `http` to `https`. Also, FYI, I added "non-exhaustive" because e.g. my Logitech Brio is supported, but isn't listed on that page.

One other thing you could mention is that a user can list the detected V4L2 webcams and capture cards on their system by running `ls -l /dev/video*` (but also note that many devices will show up twice in the list as two separate entries.)

Reply:
I'll include this, but I get "no such file or directory" when running this command on two separate machines.

Reply:
From a serw12:

![image](https://user-images.githubusercontent.com/7199422/190771833-81d8e628-129d-42d1-870a-2b1389d663f0.png)

Are you running them from a host with a camera connected (not a virtual machine)? What laptops/cameras?